scipy/signal/signaltools.py

```diff
@@ -133,6 +133,11 @@ def correlate(in1, in2, mode='full', method='auto'):
            of which is faster (default).  See `convolve` Notes for more detail.
 
            .. versionadded:: 0.19.0
+    return_lags : bool, optional
+        Returns a tuple (correlation, lags) for cross-correlation. Lags can
+        be indexed with the np.argmax of the correlation to return the lag.
+
+        Infers the desired use of the 'fft' method in 'full' mode.
```

Comment:
Why should this be the case? I think that if we're going to offer the functionality of returning lags, it should be for any combination of the `method` and `mode` arguments.